### PR TITLE
Develop fixes tested on IPCI and RMD

### PR DESCRIPTION
**Pull request by cebarboza**
- Bug fixes
- RMD passing
- eventCohorts 101 to 114
- Submitting for review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,66 +25,89 @@ Analytic software to perform large-scale distributed analysis of patients with A
 
 # Requirements
 
-
-A database in Common Data Model version 5 in one of these platforms: SQL Server, Oracle, PostgreSQL, IBM Netezza, Apache Impala, Amazon RedShift, Google BigQuery, or Microsoft APS.
```
Comment (rossdwilliams) [inline README.md:21]: OMOP

File: inst/extras/CodeToRun.R [hunk 2]
```diff
@@ -74,23 +76,23 @@ cohortTable <- "alopecia_ehden"
...
-outputFolder <- "results"
+outputFolder <- here::here("results")
 
 
 #choose analysis to run
 instantiateCohorts <- TRUE
-runDiagnostics <- TRUE
-
+runDiagnostics <- FALSE
+runPatternAnalysis <- TRUE
```
Comment (rossdwilliams) [inline inst/extras/CodeToRun.R]: default to false

File: inst/extras/CodeToRun.R
Comment (rossdwilliams) [inline inst/extras/CodeToRun.R:1]: why did the extras get moved to inst?

**Review by rossdwilliams — APPROVED** [review]

File: inst/extras/CodeToRun.R
Comment (cebarboza) [reply inst/extras/CodeToRun.R:1]: I got a warning message in RMD check that extras was not a default folder for installation.

Comment (rossdwilliams) [reply inst/extras/CodeToRun.R:1]: okay great, thats fine

**Pull request merged**